src/uct/ib/ud/base/ud_iface.c

```diff
@@ -366,6 +381,9 @@ ucs_status_t uct_ud_iface_complete_init(uct_ud_iface_t *iface)
         goto err_unset_event_handler;
     }
 
+    iface->async.cbq_elem.cb = uct_ud_iface_async_handler_remove;
```

Comment:
pls move this to earlier phase in iface creation - to where all other fields are initialized